nx.json

```diff
@@ -3,7 +3,8 @@
   "extends": "nx/presets/npm.json",
   "targetDefaults": {
     "build": {
-      "cache": true
+      "cache": true,
+      "dependsOn": ["^build"]
```

Comment:
👍🏼 I also added the same thing in my PR but then did not try to run lerna, instead continued with nx.